torchdynamo/variables/torch.py

```diff
@@ -244,6 +246,20 @@ def fake_softmax(input):
 
         return variables.LambdaVariable(fake_softmax, **options)
 
+    def _call_cross_entropy_loss(self, tx, args, kwargs, options):
```

Comment:
Perhpas assert args/kwargs are empty?  Are there possible args we need to pass to `torch.nn.functional.cross_entropy`

Reply:
Yes, great call, thanks for forcing me to take a closer look - the args are:

functional: `input, target, weight=None, size_average=None, ignore_index=- 100, reduce=None, reduction='mean', label_smoothing=0.0`

non functional ctor: `weight=None, size_average=None, ignore_index=- 100, reduce=None, reduction='mean', label_smoothing=0.0`

non functional `loss` call: `input, target, optional_output`

so it really should be shuffled, much the same way that you take `dim` from softmax.